include/aspect/material_model/steinberger.h

```diff
@@ -274,6 +274,7 @@ namespace aspect
         std::vector<double> conductivity_reference_temperatures;
         std::vector<double> conductivity_exponents;
         std::vector<double> saturation_scaling;
+        std::vector<double> prefactors;
```

Comment:
```suggestion
        std::vector<double> viscosity_prefactors;
```
Can you also move this variable down a few lines and add documentation that described what it represents. The variable is currently lumped with other variables related to thermal conductivity.